docs/content/TextArea.mdx

```diff
@@ -0,0 +1,187 @@
+---
+componentId: textarea
+title: TextArea
```

Comment:
Is textarea one or two words?

Reply:
In the React element types, they call it "TextArea". I also thought the CamelCasing looked funny here 🤷‍♂️

Reply:
I think it should be two words really, but HTML has perhaps conditioned us to believe it's one? I went with `TextArea`  because 1) If they are two words, pascalcase makes sense. 2) We call a similar component `TextInput`. 

Not precious either way. As @mperrotti pointed out, React internally refers to it in PascalCase too.